frontend/packages/dev-console/src/components/topology/application-panel/ApplicationGroupResource.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { K8sResourceKind, referenceFor } from '@console/internal/module/k8s';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import TopologyApplicationResourceList from './TopologyApplicationList';
+
+const MAX_RESOURCES = 5;
+
+export type ApplicationGroupResourceProps = {
+  title: string;
+  resourcesData: K8sResourceKind[];
+  group: string;
+};
+
+const ApplicationGroupResource: React.FC<ApplicationGroupResourceProps> = ({
+  title,
+  resourcesData,
+  group,
+}) => {
+  return !_.isEmpty(resourcesData) ? (
```

Comment:
You should move the file in one commit before making changes so that it doesn't show up as a new file.

Furthermore you can simplify this component by removing the `return` statement since you're already making other changes.

Reply:
simplified the component.